src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -804,15 +806,15 @@ public void execute() throws GitException, InterruptedException {
     public void clean(boolean cleanSubmodule) throws GitException, InterruptedException {
         reset(true);
 	String cmd = "-fdx";
-	if (cleanSubmodule) cmd = "-ffdx";
-
+    if (cleanSubmodule) cmd = "-ffdx";
+    
```

Comment:
No trailing whitespace on lines, please.  The current whitespace setup is already bad enough without adding more cases of whitespace issues.

Reply:
ditto